### PR TITLE
Add watchOS test apps for GDT and GDTCCT with compiled interface controller

### PR DESCRIPTION
**Pull request by doudounan**
- Add watchOS test apps for GDT and GDTCCT

- Similar interfaces as iOS test apps

### CODE REVIEW

File: Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -600,78 +539,30 @@
...
 		};
 		515694A423F4965200E7C350 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = EQHXZ8M8AV;
```
Comment (mikehaney24) [inline Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj]: CODE_SIGN_IDENTITY should always be "-", and DEVELOPMENT_TEAM should be empty.

File: Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 50;
```
Comment (mikehaney24) [inline Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj]: Can you explain why we're altering the Example/watchOSSample app?

File: GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTWatchOSTestAppWatchKitExtension/InterfaceController.swift [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    // This method is called when watch view controller is no longer visible
+    super.didDeactivate()
+  }
+
+  @IBAction func generateDataEvent(sender: AnyObject?) {
+    print("Generating data event")
+    let transportToUse = transport
+    let event: GDTCOREvent = transportToUse.eventForTransport()
+    let testMessage = FirelogTestMessageHolder()
+    testMessage.root.identifier = "ios_test_app_data_event"
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTWatchOSTestAppWatchKitExtension/InterfaceController.swift]: maybe change these (here and below) to say watchos_test...

File: GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTWatchOSTestAppWatchKitExtension/proto/flltest.proto [hunk 4]
```diff
@@ -0,0 +1,48 @@
+/*
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTWatchOSTestAppWatchKitExtension/proto/flltest.proto]: I'm not sure we need another copy of the proto in GDTCCT's directory--it's already in there.

File: Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj (hunk 2 above)
Comment (doudounan) [reply Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj]: The modification of watchOSSample in Example was made by mistake when I searched for reference. I will discard all the changes in this file.

File: Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -16,8 +16,6 @@
 		5156949123F4965100E7C350 /* NotificationController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5156949023F4965100E7C350 /* NotificationController.swift */; };
 		5156949323F4965100E7C350 /* ComplicationController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5156949223F4965100E7C350 /* ComplicationController.swift */; };
 		5156949523F4965200E7C350 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 5156949423F4965200E7C350 /* Assets.xcassets */; };
-		515694AF23F4A3D200E7C350 /* NotificationService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 515694AE23F4A3D200E7C350 /* NotificationService.swift */; };
```
Comment (chliangGoogle) [inline Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj]: Please revert this, this will remove messaging functions.

Comment (doudounan) [reply Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj]: The modification of watchOSSample in Example was made by mistake when I searched for reference. I will discard all the changes in this file.

File: GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestApp.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -0,0 +1,705 @@
...
+			files = (
+				9505C3262405B30E00534CF2 /* GDTWatchOSTestAppWatchKitApp.app in Embed Watch Content */,
+			);
+			name = "Embed Watch Content";
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXCopyFilesBuildPhase section */
+
+/* Begin PBXFileReference section */
+		12A683F830B22F5108F84577 /* Pods-GDTWatchOSTestAppWatchKitExtension.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-GDTWatchOSTestAppWatchKitExtension.debug.xcconfig"; path = "Target Support Files/Pods-GDTWatchOSTestAppWatchKitExtension/Pods-GDTWatchOSTestAppWatchKitExtension.debug.xcconfig"; sourceTree = "<group>"; };
```
Comment (chliangGoogle) [inline GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestApp.xcodeproj/project.pbxproj]: make sure you run `pod deintegrate` before committing so project.pbxproj is clean without installing pods information.

**Pull request merged**